### ISSUE TITLE
Convert all existing dandiset metadata to new schema and test with UI

### ISSUE THREAD

**Issue by mgrauer** [dandi/dandiarchive#458]
@AlmightyYakob will be updating the UI and the metadata editor to deal with the schema changes.

Take the [conversion script](https://github.com/dandi/dandi-core/pull/4/files#diff-f561ceca105361019955ac383cdcbeb5) from the schema update and convert all existing draft dandiset metadata from production, then try these out with the UI and metadata editor to search for issues.

You can also use this converted metadata to test out the publish process where you've included name, description, and contributor.

**Comment by satra** [comment]
note the conversion script works on the json version of the dandiset metadata key stored in girder. the json object has to look like:

```
{
  "dandiset": {
    "age": {
      "maximum": "None",
      "minimum": "14 days 00:00:00",
      "units": "TODO"
    },
    "contributors": [
...
```

**Comment by AlmightyYakob** [comment]
Probably good to link the conversion script in the new repo:
https://github.com/dandi/schema/blob/master/scripts/convert.py

**Comment by mgrauer** [comment]
Fixed by #423 .
